rocketc/rocketc.py

```diff
@@ -230,15 +237,19 @@ def login(self, user_data):
         or creates a user to login in RocketChat
         """
         rocket_chat_user = self.search_rocket_chat_user(user_data["username"])
+        LOG.info("Login method: result search user: %s", rocket_chat_user["success"])
```

Comment:
Is there any standard to bild the logs? I mean,. in some logs below I found "Method set group...", but in this one the logs syas "Login method..." so maybe is better to make log messages standard